.github/workflows/tests.yml

```diff
@@ -102,8 +102,7 @@ jobs:
       testProjectPath: tests/Aspire.Templates.Tests/Aspire.Templates.Tests.csproj
       testSessionTimeoutMs: 1200000
       testHangTimeout: 12m
-      # append '.' to the name so only the test class with exactly that name is run
-      extraTestArgs: "--filter \"(quarantined!=true)&(FullyQualifiedName~Aspire.Templates.Tests.${{ matrix.shortname }}.)\""
+      extraTestArgs: "--filter-not-trait quarantined=true --filter-class Aspire.Templates.Tests.${{ matrix.shortname }}"
```

Comment:
these are implicitly anded? I guess we would notice if not

Reply:
it would be great if you could check test counts before and after this change, before we merge the PR..!

Reply:
Yup sure.

Reply:
I made a quick test and yes they are ANDed.